.travis.yml

```diff
@@ -11,8 +11,18 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
+env:
+  global:
+    - SONAR_HOST_URL=https://sonarcloud.io
+    - GITHUB_API_URL=https://api.github.com
+    - PROJECT_KEY=ioet_bpm-people-api
+    - ORGANIZATION=ioet
+    - secure: R6cSl4b9QBM990ReJB9KU9gWqHKTER9gk/f2fdoCkaa3cg4Q4kp8LzrsXGsUY0qAskWTr00sDOpdHLv1yQfloAOZVAA6kX3SuEC+Xu3N6dd+j3ozBxVx+RUCednHpkhfG4TPotvHmBpTDCRiN7mKMxAm2gg+Vu36OvroKeBzRyRPdi+5GB3OThzdulhfQvQtg7xsFCdajERxbABy3uaBpt6gOdAXfLOM1bgvReIwssIZI2zTvFjnVoZ95MrSIRzAHrzIrzlAcaonY4iTgAfWaGs/3A3ToXTybycNrn3f/RWeMFHOjFkUJ07RYhV5opeYij3lUy6FpVH6eULrOW2nOnUK03UaVXaSn/EYXykecB/7Pi3r2TzrSHvEK8DQuxTC03PqUg4Uo/GnN6DCpdBOb5LC6SSIfHvq2SPMybHxJBNb4Rh/yjQJ3ZCRbR61kQr5eza7tpMWfaOQ9B+iR66EYyPLiiZe8igbW1rsycahournuXiwicnumr2uwesbSL+40/xxWlofvyAnDce+yfp1XfBmnl63Gl7u8w12sDOPDbWoY20BPkfsx596KXRuGxKBosb4PPnY//vNbSUB7iHb0X8IOEe8WHDgkm0HCTsC3CIdLpJd3i0SQBQXVESz4C9+pWqrGadCjHSl7pAlEBacm8xMpFDXJ3dj35b8TmqOfBo=
```

Comment:
Do these variables differ from branch to branch? If not I think we can add them in the repository settings instead of the travis.yml file as suggested in the [travis docuementation](https://docs.travis-ci.com/user/environment-variables/).
There we also export some Variables for the AWS upload and the Codecov integration.
I think it's nice to have all of them in the same place.

Reply:
Oh I see, everyone should use it's own token? Or do we create one token per project?

Reply:
@oemel09 I was thinking one token per project to reduce the impact if a single token becomes compromised, but yes, I will look into the possibility of moving these environment variables out of the code base and into the repository's configuration.